src/responsive-layout.tsx

```diff
@@ -2,24 +2,40 @@ import * as React from 'react';
 import classnames from 'classnames';
 import {getPrefixedDataAttributes} from './utils/dom';
 import * as styles from './responsive-layout.css';
+import {ThemeVariant, useIsInverseVariant} from './theme-variant-context';
 
 import type {DataAttributes} from './utils/types';
 
 type Props = {
     children: React.ReactNode;
     fullWidth?: boolean;
     className?: string;
+    isInverse?: boolean;
+    backgroundColor?: string;
     dataAttributes?: DataAttributes;
 };
 
-const ResponsiveLayout: React.FC<Props> = ({children, className, fullWidth, dataAttributes}) => {
+const ResponsiveLayout: React.FC<Props> = ({
+    children,
+    isInverse = false,
+    backgroundColor,
+    className,
+    fullWidth,
+    dataAttributes,
+}) => {
+    const isInverseOutside = useIsInverseVariant();
     return (
-        <div
-            className={classnames(styles.container, className)}
-            {...getPrefixedDataAttributes(dataAttributes)}
-        >
-            <div className={fullWidth ? styles.fullWidth : styles.responsiveLayout}>{children}</div>
-        </div>
+        <ThemeVariant isInverse={isInverse || isInverseOutside}>
```

Comment:
if `isInverseOutside` is already true, is this ThemeVariant needed?

Reply:
not really, but it's simpler this way that rendering the `ThemeVariant` conditionally